cardano-cli/test/Test/Golden/TxView.hs

```diff
@@ -173,3 +206,48 @@ golden_view_mary =
       execCardanoCLI
         ["transaction", "view", "--tx-body-file", transactionBodyFile]
     diffVsGoldenFile result "test/data/golden/mary/transaction-view.out"
+
+golden_view_alonzo :: Property
+golden_view_alonzo =
+  propertyOnce $
+  moduleWorkspace "tmp" $ \tempDir -> do
```

Comment:
Missing indent

Reply:
Is indent really needed here?

Reply:
Only if you want to follow the style doc :)

Reply:
done